### ISSUE TITLE
Specs are broken?

### ISSUE THREAD

**Issue by mittorn** [ptitSeb/gl4es#247]
[log.txt](https://github.com/ptitSeb/gl4es/files/5559349/log.txt)
How to regenerate it correctly?

**Comment by ptitSeb** [comment]
Yes, I don't know why the headers get duplicated on the generation. You need to remove the second part by hand. I tried to debug but didn't found the cause of that duplication.

**Comment by mittorn** [comment]
First one is generated in base template, second in guard template (check PR #248)
